fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/SplitHelperTest.java

```diff
@@ -462,7 +472,7 @@ public void deleteWithSplit(boolean splitLongRecords, boolean omitUnsplitSuffix)
 
     @ParameterizedTest(name = "deleteWithSplitAndVersion [splitLongRecords = {0}]")
     @BooleanSource
```

Comment:
Should this be parameterized by `unrollSingleRecordDeletes`? (or omitUnsplitSuffix, but that was already missing)